endpoint.go

```diff
@@ -1109,3 +1109,26 @@ func (c *controller) cleanupLocalEndpoints() {
 		}
 	}
 }
+
+func (ep *endpoint) setAliasIP(sb *sandbox, ip net.IP, add bool) error {
+	sb.Lock()
+	sbox := sb.osSbox
+	sb.Unlock()
+
+	for _, i := range sbox.Info().Interfaces() {
+		if ep.hasInterface(i.SrcName()) {
+			var (
+				ipNet *net.IPNet
+				err   error
+			)
+			if _, ipNet, err = net.ParseCIDR(ip.String() + "/32"); err != nil {
+				return err
+			}
```

Comment:
There are many ways to skin this. The one used now has better readability and its not a performance critical path. But if you want to construct `IPNet` without the stringing and re-parsing why would you call another API. we just need a 32 bit mask. I can change it.